src/main/java/xyz/nucleoid/extras/integrations/relay/ChatRelayIntegration.java

```diff
@@ -123,59 +128,56 @@ private void tick(MinecraftServer server) {
         }
     }
 
-    private ActionResult onSendChatMessage(ServerPlayerEntity player, String content) {
+    private void onSendChatMessage(MessageSender sender, String content) {
         var body = new JsonObject();
 
-        var profile = player.getGameProfile();
-
         var senderRoot = new JsonObject();
-        senderRoot.addProperty("id", profile.getId().toString());
-        senderRoot.addProperty("name", profile.getName());
+        senderRoot.addProperty("id", sender.uuid().toString());
+        senderRoot.addProperty("name", sender.name().getString());
```

Comment:
When using Stimuli 0.3.0, it would be good use the `player` again for this name- otherwise this sender name is the player display name rather than profile name